scripts/measurements/tools_run.sh

```diff
@@ -8,37 +8,34 @@
 PATH=$PATH:/opt/bro/bin
 export PATH
 
+truncate -s 0 performance.log
```

Comment:
It would be better to put all the log files in BITS-Darshini/data/log/ directory. It is also better to create that directory if it does not exist. Assuming the existence of that directory is going to lead to lot of unintended errors.